src/Temporalio/Client/WorkerBuildIdVersionSets.cs

```diff
@@ -0,0 +1,57 @@
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Temporalio.Client
+{
+    /// <summary>
+    /// Represents the sets of compatible Build ID versions associated with some Task Queue, as
+    /// fetched by <see cref="ITemporalClient.GetWorkerBuildIdCompatibilityAsync"/>.
+    /// </summary>
+    /// <param name="VersionSets">The sets of compatible versions in the Task Queue.</param>
+    public record WorkerBuildIdVersionSets(IReadOnlyCollection<BuildIdVersionSet> VersionSets)
+    {
+        /// <summary>
+        /// Get the default Build ID for this Task Queue.
+        /// </summary>
+        /// <returns>That Build ID.</returns>
+        public string DefaultBuildId()
+        {
+            return this.VersionSets.Last().Default();
+        }
```

Comment:
```suggestion
        public string DefaultBuildId => DefaultSet.Default;
```

Might as well make it a property. Also can you confirm that `VersionSets` will never be empty?

Reply:
Won't be empty - but, good point I forgot to make the overall client method able to return null, which is what I'll do if the TQ has no versions.